Library/Homebrew/os/mac/xquartz.rb

```diff
@@ -95,7 +94,6 @@ def provided_by_apple?
       # remain public. New code should use MacOS::X11.bin, MacOS::X11.lib and
       # MacOS::X11.include instead, as that accounts for Xcode-only systems.
       def prefix
-        @prefix ||= Pathname.new("/usr") if OS.linux?
```

Comment:
I've just removed this for now, but I think I need to do more than that.

Assuming we want to prefer the Linuxbrew/xorg/xorg installation over anything else, something like this might make more sense (pardon my mediocre Ruby-fu):

```ruby
if OS.linux?
  @prefix ||= if Pathname.new(cellar/"xorg").exist?
    # Not sure if this case is really necessary... not all that familiar with this code ATM
    Pathname.new(ENV["HOMEBREW_PREFIX"]
  elsif Pathname.new("/usr/lib/x86_64-linux-gnu/libX11.so").exist?
    Pathname.new("/usr")
  end
end
```